package.json

```diff
@@ -37,9 +37,9 @@
         "@babel/runtime": "^7.17.9",
         "@mapbox/mapbox-gl-draw": "^1.3.0",
         "@sentry/react": "^6.19.7",
+        "@the-deep/deep-ui": "^1.6.10",
         "@togglecorp/fujs": "^1.9.3",
         "@togglecorp/re-map": "^0.1.3",
-        "@togglecorp/toggle-ui": "0.14.2-beta.10",
         "apollo-link-rest": "^0.9.0-rc.1",
         "core-js": "3.22.3",
         "csv-stringify": "5.6.5",
```

Comment:
Please remove package-lock.json file. We use yarn and already have yarn.lock file